.github/workflows/release.yml

```diff
@@ -8,26 +8,20 @@ jobs:
   create-release:
     name: Create Release
     runs-on: ubuntu-latest
-    outputs:
-      upload_url: ${{ steps.create_release.outputs.upload_url }}
     steps:
+      - uses: actions/checkout@v4
       - name: Create Release
-        id: create_release
-        uses: actions/create-release@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          tag_name: ${{ github.ref }}
-          release_name: ${{ github.ref }}
-          draft: true
-          prerelease: false
+        run: |
+          gh release create ${{ github.ref_name }} --draft --verify-tag --title ${{ github.ref_name }}
```

Comment:
The command is quite simple, so we may as well do it

Here are the results of this revision:
https://github.com/kennethloeffler/rojo/actions/runs/8657764221 (I had to re-run the macOS ARM job because I got rate limited, but the rest of the artifacts are visible in the previous run)
https://github.com/kennethloeffler/rojo/releases/tag/v7.4.0-release-workflow-testing-2